go/porcelain/deploy.go

```diff
@@ -91,7 +91,7 @@ type DeployOptions struct {
 
 	files             *deployFiles
 	functions         *deployFiles
-	functionSchedules []models.FunctionSchedule
+	functionSchedules []*models.FunctionSchedule
```

Comment:
The change to pointers here is just so that it matches the [`FunctionSchedules` type from `DeployFiles`](https://github.com/netlify/open-api/blob/master/go/models/deploy_files.go#L37) without doing any conversion.